### ISSUE TITLE
Suggestion for tags not working

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/redmine#318]
It used to be the case that a list of existing tags would appear when you start typing in the "Add a tag" field...

**Comment by pgleeson** [comment]
Related to https://github.com/OpenSourceBrain/redmine/issues/150?

**Comment by sanjayankur31** [comment]
@pgleeson : is this still a valid bug?

**Comment by pgleeson** [comment]
Yes... but strangely I added a new tag "Rate based model" on this: https://github.com/OpenSourceBrain/WilsonCowan with no suggestions popping up, then that was the only suggestion to come up on another project...
